tssc/step_implementers/push_artifacts/maven.py

```diff
@@ -159,16 +161,16 @@ def _run_step(self, runtime_step_config):
 
         # ----- get generate-metadata items
         # Required: Get the generate-metadata.version
-        if(self.get_step_results('generate-metadata') and \
-          self.get_step_results('generate-metadata').get('version')):
+        if (self.get_step_results('generate-metadata') and
```

Comment:
doesn't this need the `\` to work?

Reply:
line continuation is "automatic" when the split is in parenthesis.